components/texas-instruments/SN74LVC245A.stanza

```diff
@@ -48,41 +48,5 @@ public pcb-component component (pkg-dwg:String) :
   make-box-symbol()
   assign-landpattern(pkg)
 
-  eval-when has-property?(VCC.voltage) :
-    val v = property(VCC.voltage)
-    property(VCC.power-pin) = true
-    property(VCC.vVxx-absmax) = [-0.5 6.5]
-    property(VCC.vVxx-op) = [1.65 3.6]
-    ; TODO: Evaluate max or min or both voltage cases
-    property(VCC.iOH) = PWL([[1.65 -4.0e-3], [2.30 -8.0e-3], [2.70 -12.0e-3], [3.0 -24.0e-3], [3.6 -24.0e-3]])[v[0]]
-    property(VCC.iOL) = PWL([[1.65  4.0e-3], [2.30  8.0e-3], [2.70  12.0e-3], [3.0  24.0e-3], [3.6  24.0e-3]])[v[0]]
```

Comment:
Did you mean to delete this chunk and the commented out chunk?

Reply:
Yes - this is stale

Reply:
Agree